lib/src/value/mixin.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:meta/meta.dart';
+
+import '../callable.dart';
+import '../visitor/interface/value.dart';
+import '../value.dart';
+
+/// A SassScript mixin reference.
+///
+/// A mixin reference captures a mixin from the local environment so that
+/// it may be passed between modules.
+///
+/// {@category Value}
+@sealed
```

Comment:
As this is a new class (and so there is no BC concern on it), should it directly use a Dart 3 modifier (refs #1978) or do you prefer to have them all or none ?

Reply:
Good point, let's make this properly `final`.